tests/test_black.py

```diff
@@ -1229,12 +1234,14 @@ def test_invalid_cli_regex(self) -> None:
 
     def test_required_version_matches_version(self) -> None:
         self.invokeBlack(
-            ["--required-version", black.__version__], exit_code=0, ignore_config=True
+            ["--required-version", black.__version__, "-c", "0"],
+            exit_code=0,
+            ignore_config=True,
         )
 
     def test_required_version_does_not_match_version(self) -> None:
         self.invokeBlack(
-            ["--required-version", "20.99b"], exit_code=1, ignore_config=True
+            ["--required-version", "20.99b", "-c", "0"], exit_code=1, ignore_config=True
```

Comment:
Technically still would succeed without the dummy code, but the exit code can't tell the source. Should we assert on error message?

Reply:
On a second thought, the code is required, because otherwise the error message will be the source requirement. But we could still assert on the error to make sure to avoid mistakes in the future.